receiver/prometheusreceiver/internal/util.go

```diff
@@ -82,7 +81,8 @@ func getBoundary(metricType pmetric.MetricType, labels labels.Labels) (float64,
 	case pmetric.MetricTypeSummary:
 		val = labels.Get(model.QuantileLabel)
 		if val == "" {
-			return 0, errEmptyQuantileLabel
+			// if there's no quantile label, just return 0, which is the default
```

Comment:
Is this needed to be able to handle series without quantile labels?  I thought sum and count never reached this point.

Reply:
We call this in `addSeries` here and propagate the error, which causes it for fail the entire transation. An alternative would be to keep this logic as is, but handle the error differently in `addSeries`.

Reply:
I would love to handle the error differntly in addSeries.  We definitely shouldn't fail the entire scrape if there is a single bad series (assuming that is the current behavior).

Reply:
That is indeed the current behaviour. Changing that behaviour is straightforward, but we have a lot of openmetrics validation tests which depend on this behaviour. I'll see if I can find a way to make this a relatively painless change.

Reply:
@dashpole I created a separate PR for this change: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/24032. Can you have a look?